src/main/java/org/altbeacon/beacon/service/scanner/CycledLeScanner.java

```diff
@@ -169,6 +170,13 @@ public void destroy() {
         mScanThread.quit();
     }
 
+    protected boolean canScan() {
+        if (!mContext.getPackageManager().hasSystemFeature(PackageManager.FEATURE_BLUETOOTH_LE)) {
+            return false;
+        }
+        return true;
+    }
```

Comment:
This can be simplified to just:

```java
protected boolean canScan() {
  return mContext.getPackageManager().hasSystemFeature(PackageManager.FEATURE_BLUETOOTH_LE);
}
```